code/modules/projectiles/projectile.dm

```diff
@@ -850,6 +850,9 @@
 
 //Spread is FORCED!
 /obj/projectile/proc/preparePixelProjectile(atom/target, atom/source, modifiers, spread = 0)
+	if(modifiers && !islist(modifiers))
```

Comment:
```suggestion
	if(!isnull(modifiers) && !islist(modifiers))
```

I know it's unlikely, but while you're doing a test like this you might as well cover your bases and don't let FALSE/0 through. Plus, it's more explicit.